content/en/blog/_posts/2020-12-08-kubernetes-release-1.20.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: blog
+title: 'Kubernetes 1.20: The Raddest Release'
+date: 2020-12-08
+slug: kubernetes-1-20-release-announcement
+---
+
+**Authors:** [Kubernetes 1.20 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.20/release_team.md)
+
+We’re pleased to announce the release of Kubernetes 1.20, our third and final release of 2020! This release consists of 44 enhancements: 11 enhancements have graduated to stable, 15 enhancements are moving to beta, and 16 enhancements are entering alpha.
```

Comment:
11 + 15 + 16 != 44?
I believe 2 enhancements were tracked as deprecation. 
Not sure if it matters, but I feel like someone could think this was miswritten.

Reply:
Yea I can see that if someone doesn't know about deprecations. @jeremyrickard thoughts?

Reply:
Yeah, I believe this is correct. 2 deprecations in that number.

Reply:
It's probably worth saying N enhancements, breakdown, and then talk about the two that are deprecations?

I think we didn't actually track the dockershim deprecation, so it's probably actually 45, 3 of which are deprecation?

Reply:
I double checked the last few releases. Deprecations were not counted. I'll update the enhancement number.